docs/protocol/bugs.md

```diff
@@ -1,4 +1,5 @@
 # Bugs
 
+When creating a Layer 1 (L1) protocol chain, taking security seriously is of utmost importance. L1 protocols are the backbone of the blockchain ecosystem, and any security vulnerabilities within the L1 can lead to a range of potential consequences, from financial loss to reputation damage to even the collapse of the entire network. As such, security is an essential aspect of developing any L1 protocol chain, and security must be considered at every step of the development process. We did conduct an audit with [Informal System](https://github.com/informalsystems/audits/blob/main/Evmos2021Q4/informal-evmos-report-2021q4.pdf).
```

Comment:
I'd link to the audits page instead of linking to a single specific audit

Reply:
Okay